climpred/tests/test_PredictionEnsemble_math.py

```diff
@@ -0,0 +1,307 @@
+import numpy as np
+import pytest
+import xarray as xr
+from xarray.testing import assert_equal
+
+from climpred import HindcastEnsemble, PerfectModelEnsemble
+from climpred.classes import PredictionEnsemble
+from climpred.exceptions import VariableError
+
+ALLOWED_TYPES_FOR_MATH_OPERATORS = [
```

Comment:
Maybe this goes into `climpred.constants` since it's used twice now.

Reply:
in classes it contains `type(self)` and I have circular import problems when trying to move checks to `checks`